experimental/await-ops.md

```diff
@@ -0,0 +1,22 @@
+# [Await.ops](https://github.com/tc39/proposal-await.ops)
+
+## AwaitExpression
+
+```js
+interface AwaitExpression <: Expression {
+    type: "AwaitExpression";
+    argument: Expression;
```

Comment:
```suggestion
extend interface AwaitExpression {
```
An editorial update. All experimental proposals are filed as a diff to current spec.

Reply:
done thanks,